### PR TITLE
Button: Renaming BaseButton to Actionable, extending Button from Actionable and removing content and icon from Actionable types

### PR DESCRIPTION
**Pull request by khmakoto**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

This PR makes `Button` extend a more bare bones `Actionable` component that has some basic styling and lacks the content and icon slots to make it more similar to the native `button` component and improve perf.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9436)

### CODE REVIEW

**Comment by khmakoto** [comment]
The screener failures are on purpose because we're changing the styling of the `BaseButton`/`Actionable`.

**Comment by khmakoto** [comment]
@dzearing how do the screener changes look to you? Do they resemble what you had thought for the `Actionable` component?

**Comment by dzearing** [comment]
@khmakoto ping on this one; conflicts, build errors. LMK when it's ready :) Getting stale

File: packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx [hunk 1]
```diff
@@ -958,7 +958,7 @@ export class BasePickerListBelow<T, P extends IBasePickerProps<T>> extends BaseP
           {this.getSuggestionsAlert(classNames.screenReaderText)}
           <div className={classNames.text}>
             <Autofill
-              {...(inputProps as any)}
+              {...inputProps as any}
```
Comment (khmakoto) [inline packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx]: Prettier change.

**Comment by khmakoto** [comment]
@dzearing I think it's ready. It's above the tolerance level for bundlesize but I think it should be fine since my Stack PR last week took off 2K from the bundle size.

**Pull request merged**